packages/dds/tree/src/test/shared-tree/fuzz/fuzzEditGenerators.ts

```diff
@@ -261,7 +300,7 @@ function getRandomPlace(tree: ISharedTree, random: IRandom): UpPath {
 			return { ...parentPath, parentIndex: parentPath.parentIndex + 1 };
 		case "belowNode":
 			cursor.free();
-			return { parent: parentPath, parentField: testerKey, parentIndex: 0 };
+			return { parent: parentPath, parentField: random.pick(fieldKeys), parentIndex: 0 };
```

Comment:
```suggestion
			return { parent: parentPath, parentField: sequenceField, parentIndex: 0 };
```

Reply:
changed to hardcoded field `sequenceField`